pom.xml

```diff
@@ -53,6 +53,11 @@
             <version>8.0</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.eclipse.persistence</groupId>
+            <artifactId>org.eclipse.persistence.jpa</artifactId>
```

Comment:
with this dependency we're now bundling an entire copy of Eclipselink inside the application, which will cause the app to start slower, have a larger disk/mem footprint, and potentially cause classloading conflicts with the one included by default with the Liberty runtime.

Instead, this can be changed to a `provided` scope dependency, and then you can access the EclipseLink API bundled with Liberty by opting into third-party API in server.xml with:
```xml
    <webApplication id="Portfolio" name="Portfolio" location="Portfolio.war" contextRoot="/portfolio">
        <classloader apiTypeVisibility="+third-party"/>
    </webApplication>
```

Reply:
This needs to be fixed.  Do we even use/need any EclipseLink API here, @andrewdes?

Should be a straightforward JPA dependency?

Reply:
@yeekangc yes there is one usage of an EclipseLink annotation (`@org.eclipse.persistence.annotations.CascadeOnDelete`)